### ISSUE TITLE
Name change

### ISSUE THREAD

**Issue by rafnov** [maxistar/TextPad#143]
In my language app name still says Prosty edytor tekstu which has nothing to do with Textpad, but means just Simple text editor. It should be changed.

**Comment by maxistar** [comment]
You mean the Name of the application should not be translated? I will check this out, thanks tor the suggestion.

**Comment by rafnov** [comment]
Yes, it shouldn't, as by own original name it is usually easier to recognize certain particular apps.
Textpad is original - one app found on F-droid.
Prosty edytor tekstu - 2 apps found.
Simple text editor - again 1 app found but it's not even Textpad 😆
I think the best is to leave the original name, but the description should begin with the best words describing: "simple text editor". And as of F-droid is translated it should give proper app search even when searched in native languages.
